core/amber/src/main/scala/edu/uci/ics/texera/web/resource/dashboard/workflow/WorkflowExecutionsResource.scala

```diff
@@ -71,4 +81,32 @@ class WorkflowExecutionsResource {
         .toList
     }
   }
+
+  /** Sets a single execution's bookmark to the payload passed in the body. */
+  @PUT
+  @Path("/set_execution_bookmark")
+  @Consumes(Array(MediaType.APPLICATION_JSON))
+  def setExecutionIsBookmarked(
+      request: ExecutionBookmarkRequest,
+      @Auth sessionUser: SessionUser
+  ): Unit = {
+    validateUserCanAccessWorkflow(request.wid, sessionUser.getUser.getUid)
+    getExecutionById(request.eId) match {
+      case Some(execution) => {
+        val isBookmarked: Byte = if (request.isBookmarked) 1 else 0
+        execution.setBookmarked(isBookmarked)
```

Comment:
```suggestion
        execution.setBookmarked((Byte) request.isBookmarked?1:0)
```

Reply:
So scala doesn't like this (it wouldn't compile this way). Reading online, it prefers calling methods to cast.
So I opted for:
`        execution.setBookmarked((if (request.isBookmarked) 1 else 0).toByte)`

Also ternary's in scala aren't "?" and ":", those are in java apparently